src/client/Client.cc

```diff
@@ -3534,25 +3534,20 @@ static int adjust_caps_used_for_lazyio(int used, int issued, int implemented)
   if (!(implemented & CEPH_CAP_FILE_LAZYIO))
     return used;
 
+  used |= CEPH_CAP_FILE_LAZYIO;
```

Comment:
Isn't this changing the logic in this function? Before, LAZYIO would only be set in the returned flags iff any of Fcbl were issued. Now, it's always returned even if none of them are issued.

Reply:
Fixed it. Thanks.